File: Python/FindAllNumbersDisappearedInArray.py
def findAllNumbers(nums):
    outputList = []

    for n in nums:
        i = abs(n)-1
        nums[i] = -1 * abs(nums[i])

    for i in range(len(nums)):
        if nums[i]>0:
            outputList.append(i+1)

    return outputList

File: Javascript/FindAllNumbersDisappearedInArray.js
var findAllNumbers = function(nums){
    let outputList = []

    for(let n of nums){
        let i = Math.abs(n)-1
        nums[i] = -1 * Math.abs(nums[i])
    }

    for(let i=0;i<nums.length;i++){
        if(nums[i]>0){
            outputList.push(i+1)
        }
    }

    return outputList
}